Testing/Tests/tag_expansion_test.py

```diff
@@ -0,0 +1,69 @@
+"""
+This class ensures that TagExpansion.py correctly expands all vector tags in a mesh.
+"""
+
+import os
+import pymoab
+import pytest
+
+from PythonTool.TagExpansion import get_tag_lists, create_directory, expand_vec_tag
+
+# Initialize a PyMOAB core instance and load in the h5m file.
+data_file = "Testing/SampleData/photon_flux_mesh.h5m"
+mb = pymoab.core.Core()
+mb.load_file(data_file)
+
+
+def test_get_tag_lists():
+    """
+    Ensure this function correctly retrieves all vector tags on an element.
+    """
+    elements, scalars, vectors = get_tag_lists(mb, "hex", pymoab.types.MBHEX)
+    assert len(vectors) == 2
+
+
+def test_get_tag_lists_exception():
+    """
+    Ensure this function correctly detects when there are none of a specific element in the mesh.
+    """
+    with pytest.raises(LookupError) as e:
+        get_tag_lists(mb, "tet", pymoab.types.MBTET)
+
+
+def test_create_directory():
+    """
+    Ensure this function correctly creates a directory for tag expansion.
+    """
+    dir_name = create_directory(data_file, "Testing")
+    assert os.path.isdir("Testing_database") == True
```

Comment:
Do you need to remove this directory after the assertion so it doesn't dirty the repo?